DQMOffline/L1Trigger/src/L1TTauOffline.cc

```diff
@@ -533,10 +533,21 @@ bool L1TTauOffline::matchHlt(edm::Handle<trigger::TriggerEvent> const& triggerEv
 
   trigger::TriggerObjectCollection trigObjs = triggerEvent->getObjects();
 
-  for (auto trigIndexIt = m_trigIndices.begin(); trigIndexIt != m_trigIndices.end(); ++trigIndexIt) {
+  for (vector<int>::const_iterator trigIndexIt = m_trigIndices.begin(); trigIndexIt != m_trigIndices.end();
+       ++trigIndexIt) {
     const vector<string> moduleLabels(m_hltConfig.moduleLabels(*trigIndexIt));
```

Comment:
Probably a bit cleaner:
```suggestion
  for (auto& trigIndex : trigIndexIt != m_trigIndices) {
    const vector<string> moduleLabels(m_hltConfig.moduleLabels(trigIndex));
```